packages/base/Form/src/FormLabel/styles.ts

```diff
@@ -1,55 +1,66 @@
-import type { Theme } from '@material-ui/core/styles'
-import { createStyles } from '@material-ui/core/styles'
-import { alpha } from '@toptal/picasso-shared'
-
-export default ({ palette }: Theme) =>
-  createStyles({
-    root: {
-      color: palette.grey[400],
-      display: 'block',
-      marginBottom: '0.5em',
-      lineHeight: '1em',
-    },
-
-    disabled: {
-      color: alpha(palette.grey[400], 0.48),
-    },
-
-    medium: {
-      fontSize: '0.875rem',
-    },
-
-    large: {
-      fontSize: '1rem',
-    },
-
-    asterisk: {
-      marginRight: '0.3125em',
-      color: palette.error.main,
-    },
-
-    inline: {
-      display: 'inline-block',
-      marginBottom: 0,
-
-      '& $medium': {
-        fontSize: '0.8125rem',
-        verticalAlign: 'top',
-      },
-
-      '& $asterisk': {
-        fontSize: '0.8125rem',
-        verticalAlign: 'top',
-      },
-    },
-
-    horizontalLayout: {
-      display: 'flex',
-      alignItems: 'center',
-      marginBottom: 0,
-    },
-    alignmentTop: {
-      alignItems: 'start',
-      paddingTop: '0.5rem',
-    },
-  })
+import { twJoin } from '@toptal/picasso-tailwind-merge'
+
+import type { FieldLayout } from '../FieldsLayout/FieldsLayoutContext'
+import type { Alignment, Size } from './FormLabel'
+
+export const classesBySize: Record<Size, string> = {
+  medium: 'text-[0.875rem]',
+  large: 'text-[1rem]',
+}
+
+type GetRootClassesOptions = {
+  disabled?: boolean
```

Comment:
NIT (to be aligned with `isInline`)

```suggestion
  isDisabled?: boolean
```

Reply:
I usually prefer it to be consistent with native APIs, in this case `disabled` is very similar to the [HTML disabled attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled). The `is` prefix seems unnecessary given Typescript types to boolean and the overall semantic meaning of the prop

Reply:
It is true that `isInline` is incosistent with `disabled`, but I would prefer to change `isInline` to `inline` to keep usage simple:

```js
const obj = {  inline: true }
// The code above looks a bit more terse than the one below
const obj = { isInline: true }

// JSX syntax is a bit more clearer
const el = <Component inline />
// vs
const el = <Component isInline />
```

Reply:
I'd like to keep `isInline` as we have `inline` prop in `FomrLabel` component and I want it to be clear that it is a different thing. The same goes for `disabled`. I want to keep it without change to be consistent with the component API.